src/settings/mod.rs

```diff
@@ -72,20 +78,34 @@ pub struct Sendgrid {
     pub key: String,
 }
 
+#[derive(Debug, Default, Serialize, PartialEq)]
+pub struct SqsUrl(pub String);
+
+impl<'d> Deserialize<'d> for SqsUrl {
```

Comment:
Actually, there might be an argument for keeping these `impl Deserialize` bits in a dedicated `deserialize` module. Not all of the types we're deserializing are exclusive to `Settings`, so if we want to have the same level of code re-use in the function bodies, it might be easier to put them in one place.

(but also, I might change my mind about that when I see what it actually looks like, not sure yet)

Reply:
I actually think it's good to have the `Deserialize` implementation right next to the `struct` definition, makes it easy to find it, if it's standardized that way, all the implementations together with the definition etc.

I think after I do this for more types we will be able to see better if it gets too confusing.

Reply:
Thinking about it now, if that is good it could even become a standard we apply for everything. Always have the `impl` blocks together with the `struct` definitions? Anyways, we'll see, hehe.

Reply:
I agree that the `impl` and `struct` should be next to each other, but I'm not sure that place should be the `settings` module.